revno: 554
committer: Andrew Psaltis <ampsaltis@gmail.com>
branch nick: 1.6
timestamp: Mon 2010-03-29 13:19:08 -0400
message:
  Fixed typo in EAP-fast template, re-resolving bug 261804.

=== modified file 'encryption/templates/eap'
--- encryption/templates/eap	2009-03-01 02:52:27 +0000
+++ encryption/templates/eap	2010-03-29 17:19:08 +0000
@@ -11,10 +11,10 @@
 	proto=RSN WPA
 	pairwise=CCMP TKIP
 	group=CCMP TKIP
-	key-mgmt=WPA-EAP
+	key_mgmt=WPA-EAP
 	eap=FAST
 	identity="$_USERNAME"
 	password="$_PASSWORD"
 	phase1="fast_provisioning=1"
-	pac-file="$_PAC_FILE"
+	pac_file="$_PAC_FILE"
 }
